app/uk/gov/hmrc/gform/testonly/TestOnlyController.scala

```diff
@@ -907,4 +932,53 @@ class TestOnlyController(
         )
     }
 
+  private val restoreOption1 = "restoreOption1"
+  private val restoreOption2 = "restoreOption2"
```

Comment:
There must be some meaningful names rather then `restoreOption1` and `restoreOption2`.